company-profiles/chess.md

```diff
@@ -0,0 +1,27 @@
+# Chess
+
+## Company blurb
+
+Chess.com is a global team of passionate chess fans and developers. You can meet us here and read about our company here. We're always looking for great people to join the team. We have a healthy work environment focused on well-being, respect, and memes/emotes. We communicate with Slack and positive energy beams.
```

Comment:
You say here, but havent hyperlinked to anything was that a typo or is the blurb the `here`

Reply:
yeah I missed the hyperlinks, thanks!